restricted_site_access.php

```diff
@@ -248,6 +286,11 @@ protected static function is_restricted() {
 
 		$blog_public = get_option( 'blog_public', 2 );
 
+		// If rsa_mode==enforce we override the rsa_options.
```

Comment:
Why is this chunk here? Seems unrelated and like it should be correctly handled after https://github.com/10up/restricted-site-access/commit/0487b14d1c04d6ebf1774e2661080d0c052df5ac?

Reply:
@helen I removed the logging in https://github.com/10up/restricted-site-access/pull/29/commits/8165c9c281909ee2be092026030776cd9fa5f91c. 

Reverted unrelated changes in https://github.com/10up/restricted-site-access/pull/29/commits/f6e27c86cc3c327137ea1bec02691744a036386f